Converters/Revit/Speckle.Converters.RevitShared/Helpers/RevitMaterialCacheSingleton.cs

```diff
@@ -17,6 +17,8 @@ public class RevitMaterialCacheSingleton
   /// </summary>
   public Dictionary<string, Dictionary<string, RenderMaterialProxy>> ObjectRenderMaterialProxiesMap { get; } = new();
 
+  public Dictionary<string, string> ObjectIdAndMaterialIndexMap { get; } = new();
```

Comment:
add POC comment explaining the hack, and why it's needed now, and how it will go away